lib/reek/configuration/configuration_file_finder.rb

```diff
@@ -18,16 +18,11 @@ class ConfigFileException < StandardError; end
     module ConfigurationFileFinder
       module_function
 
-      def find_and_load(params = {})
+      def find_and_load(**params)
         load_from_file(find(params))
       end
 
-      # FIXME: switch to kwargs on upgrade to Ruby 2 and drop `params.fetch` calls:
-      # def find(options: nil, current: Pathname.pwd, home: Pathname.new(Dir.home))
-      def find(params = {})
-        options = params.fetch(:options) { nil                    }
-        current = params.fetch(:current) { Pathname.pwd           }
-        home    = params.fetch(:home)    { Pathname.new(Dir.home) }
+      def find(options: nil, current: Pathname.pwd, home: Pathname.new(Dir.home))
```

Comment:
This is already fixed in master: https://github.com/troessner/reek/blob/master/lib/reek/configuration/configuration_file_finder.rb#L25